demo-targets/old-typo3/Chart.yaml

```
@@ -0,0 +1,20 @@
+# SPDX-FileCopyrightText: 2021 iteratec GmbH
+#
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: v2
+version: v3.1.0-alpha1
+appVersion: "v1.0"
```

Maybe we could make this match the typo3 version used (v9.5.26).
This is used as the default image tag and it would be nice to have this match the actual typo3 version of the chart & image.

```suggestion
appVersion: "v9.5.26"
```

---

Yeah that is better. I'll commit it in